lib/spectests/tests/excludes.txt

```diff
@@ -22,164 +22,48 @@ clif:skip:names.wast:*         # Names file has parsing error?
 clif:skip:simd.wast:*          # SIMD not implemented
 clif:skip:simd_binaryen.wast:* # SIMD not implemented
 
-clif:fail:binary-leb128.wast:74 # Module - caught panic Any
-clif:fail:binary-leb128.wast:86 # Module - caught panic Any
-clif:fail:binary-leb128.wast:98 # Module - caught panic Any
-clif:fail:data.wast:27 # Module - caught panic Any
-clif:fail:data.wast:40 # Module - caught panic Any
-clif:fail:data.wast:50 # Module - caught panic Any
-clif:fail:data.wast:55 # Module - caught panic Any
-clif:fail:data.wast:61 # Module - caught panic Any
-clif:fail:data.wast:66 # Module - caught panic Any
-clif:fail:data.wast:83 # Module - caught panic Any
-clif:fail:data.wast:98 # Module - caught panic Any
-clif:fail:data.wast:117 # Module - caught panic Any
-clif:fail:data.wast:127 # Module - caught panic Any
-clif:fail:data.wast:132 # Module - caught panic Any
-clif:fail:data.wast:137 # Module - caught panic Any
-clif:fail:data.wast:143 # Module - caught panic Any
-clif:fail:data.wast:149 # Module - caught panic Any
-clif:fail:data.wast:154 # Module - caught panic Any
 clif:fail:data.wast:186 # AssertUnlinkable - instantiate successful, expected unlinkable
 clif:fail:data.wast:194 # AssertUnlinkable - instantiate successful, expected unlinkable
-clif:fail:data.wast:211 # AssertUnlinkable - caught panic Any
-clif:fail:data.wast:227 # AssertUnlinkable - caught panic Any
-clif:fail:data.wast:258 # AssertUnlinkable - caught panic Any
-clif:fail:data.wast:273 # AssertUnlinkable - caught panic Any
-clif:fail:elem.wast:101 # Module - caught panic Any
 clif:fail:elem.wast:143 # AssertUnlinkable - instantiate successful, expected unlinkable
 clif:fail:elem.wast:152 # AssertUnlinkable - caught panic Any
 clif:fail:elem.wast:161 # AssertUnlinkable - instantiate successful, expected unlinkable
 clif:fail:elem.wast:170 # AssertUnlinkable - instantiate successful, expected unlinkable
 clif:fail:elem.wast:178 # AssertUnlinkable - instantiate successful, expected unlinkable
```

Comment:
I reviewed some of these regressions and they seem unrelated. Most likely were false positives previously.